frontend/app/src/features/room/components/info/RoomUserInfo.tsx

```diff
@@ -0,0 +1,32 @@
+import { useGetUserInfo } from "../../api/useGetUserInfo"
+import { Text, Spinner, Avatar, HStack, VStack } from "@chakra-ui/react"
+
+type RoomUserInfoProps = {
+    user: {
+        id: string;
+        connected: boolean;
+    }
+}
+
+export const RoomUserInfo = ({ user }: RoomUserInfoProps) => {
+
+    const { id, connected } = user
+
+    const { isLoading, isError, data } = useGetUserInfo(id)
+
+    if (isLoading) return (<Spinner />)
+
+    if (isError || !data) return (<Text>Errored</Text>)
+
+    const { name, avatarUrl, email } = data.data.user
+
+    return (
+        <HStack gap="0.75rem">
+            <Avatar name={name} src={avatarUrl} />
+            <VStack alignItems="start" gap="0">
+                <Text textStyle="heading-xs">{name ?? email}</Text>
+                {connected ? <Text>Online</Text> : <Text>Offline</Text>}
```

Comment:
might be nice to have grayed out text for offline/ green text for online

Reply:
Initial change made in [8fb0ae6](https://github.com/CS3219-AY2324S1/ay2324s1-course-assessment-g03/pull/34/commits/8fb0ae6cd894901988191d3a56ec92e1601cd445) but up for further changes